### DISCUSSION TITLE
New hardware: notepad HP EliteBook 6930p

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#35] [Show and tell]
Unable to boot from naked usb+img. Ventoy+iso ok.
Wav: ko
Touchpad: ok
keyboard: ok
ethernet: not tested.
VBE: ok 1024x768 but not max resolution of the screen 1280x800, so blurry.
[hpelitebook6930pu15.txt](https://github.com/VendelinSlezak/BleskOS/files/13796511/hpelitebook6930pu15.txt)

**Comment by VendelinSlezak** [comment]
> Wav: ko

I completely rewrote HDA driver and I took into account also codec on this computer, so we will see if it will work.

**Comment by VendelinSlezak** [comment]
Would it be possible for you to test this computer again? I would like to know if HDA driver can successfully initalize this codec.

**Comment by defdefred** [comment]
No errorm but no sound.
[elitebook6930pnosound.txt](https://github.com/VendelinSlezak/BleskOS/files/13955696/elitebook6930pnosound.txt)

**Comment by VendelinSlezak** [comment]
Thank you for testing, I think this may be related to stream reset issues I encountered with some codecs, but for now I do not know how to fix it.

**Comment by VendelinSlezak** [comment]
Would you please test if Update 27 works better?

**Comment by defdefred** [comment]
Sure.
backligth is working.
no sound output.
[EB6930P_BOS27.txt](https://github.com/VendelinSlezak/BleskOS/files/14530559/EB6930P_BOS27.txt)

**Comment by VendelinSlezak** [comment]
Thank you for testing. Glad to hear that backlight works good. Hopefully in reasonable amount of time I will also finish whole Intel graphic card driver, so it will be able to set best graphic mode on this computer. Anyway, I finally found and fixed what was causing issues with playing sound in HDA driver, so I am hope if that will help this codec to work.
